airflow/bin/cli.py

```diff
@@ -283,6 +292,43 @@ def _tabulate(pools):
         log.info(_tabulate(pools=pools))
 
 
+def pool_import_helper(filepath):
+    with open(filepath, 'r') as poolfile:
+        pl = poolfile.read()
+    try:
+        d = json.loads(pl)
+    except Exception:
+        print("Invalid pool file.")
+    else:
+        try:
+            pools = []
+            n = 0
+            for k, v in d.items():
+                if isinstance(v, dict) and len(v) == 2:
+                    pools.append(api_client.create_pool(name=k,
+                                                        slots=v["slots"],
+                                                        description=v["description"]))
+                    n += 1
+                else:
+                    pass
+        except Exception:
+            pass
+        finally:
+            print("{} of {} pool(s) successfully updated.".format(n, len(d)))
+            return pools
+
+
+def pool_export_helper(filepath):
+    pool_dict = {}
+    pools = api_client.get_pools()
```

Comment:
https://github.com/apache/incubator-airflow/blob/0ba510e238ebb197bce9afccdb06b88871c4a7e6/airflow/api/client/api_client.py#L53-L55

🤔 Did you test this?

Reply:
Yes, I did test on this and unit test passed. I used the default config for the api_client here https://github.com/apache/incubator-airflow/blob/00adade4163197fcff799168398edc629f989d34/airflow/config_templates/default_airflow.cfg#L183

I think it is actually using this class https://github.com/apache/incubator-airflow/blob/da1f64cf805abaecda227fc8c43a15a92faff648/airflow/api/client/local_client.py#L40-L42